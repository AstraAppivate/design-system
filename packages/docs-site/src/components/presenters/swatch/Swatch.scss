.swatch-container {
  display: inline-block;  
  margin-right: 20px;
}

.swatch {
  width: 40px;
  height: 40px;
  overflow: hidden;
  border-radius: 3px;
}

.swatch-label {
  text-align: center;
  font-size: 12px;
  width: 100%;
  display: block;
  margin-top: 10px;
  color: #11202b;
}