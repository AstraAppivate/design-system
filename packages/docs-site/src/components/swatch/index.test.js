import React from 'react'
import renderer from 'react-test-renderer'

import Swatch from './index'

describe('Swatch', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Swatch property="test" value="test" label="test" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
