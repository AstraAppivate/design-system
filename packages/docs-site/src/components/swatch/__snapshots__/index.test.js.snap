// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Swatch renders correctly 1`] = `
<div
  className="swatch"
>
  <div
    className="swatch-block"
    style="test:test"
  />
  <h1
    className="label"
  >
    test
  </h1>
</div>
`;
